plugins/providers/virtualbox/config.rb

```diff
@@ -107,6 +107,14 @@ def cpus=(count)
         customize("pre-boot", ["modifyvm", :id, "--cpus", count.to_i])
       end
 
+      # Shortcut for setting para-Virtualization setting for the virtual machine.
+      # Calls #customize internally.
+      #
+      # @param type [String] para-Virtualization type(v5.0: legacy,minimal,hyperv,kvm)
+      def paravirtprovider=(type)
+        customize("pre-boot", ["modifyvm", :id, "--paravirtprovider", type])
+      end
+
```

Comment:
Just tested this in conjunction with VBox 5.0. Updated vagrantfile to bring up the box with KVM paravirt, box brought up with KVM just fine :+1: